Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -697,6 +704,26 @@
       }
     ]
   },
+  {
+    "Name": "Parsers",
+    "Icon": "Icons/Helpers.png",
+    "Samples": [
+      {
+        "Name": "Markdown Parser",
+        "Type": "MarkdownParserPage",
+        "About": "The Markdown Parser allows you to parse a Markdown String into a Markdown Document, and then Render it with a Markdown Renderer.",
+        "Icon": "/Assets/Helpers.png",
```

Comment:
@nmetulev do we have some designer time before 2.2 to get images for these (and other things like composition brushes)?

Reply:
We could create a cleanup PR, there are several VS complaints from other Toolkit items that need to be fixed too.